### PR TITLE
CVCharts, standard and compact

### PR DESCRIPTION
**Pull request by Mati0x**
#258

### CODE REVIEW

File: apps/web/components/Charts/ConvictionBarChart.tsx
Comment (Corantin) [inline apps/web/components/Charts/ConvictionBarChart.tsx:1]: - Put the flag as threshold legend item
- Add info tips to each legend item to tell what the percentage is refering to
- 
image

Comment (Corantin) [inline apps/web/components/Charts/ConvictionBarChart.tsx:1]: Not sure if we are suppose to see the conviction percentage here:
image

**Review by Corantin — CHANGES_REQUESTED** [review]

File: apps/web/components/Charts/ConvictionBarChart.tsx
Comment (Mati0x) [reply apps/web/components/Charts/ConvictionBarChart.tsx:1]: Good idea!

Comment (Mati0x) [reply apps/web/components/Charts/ConvictionBarChart.tsx:1]: In this scenario the proposal value should be in the right side of the chart!, yess good catch

File: apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx [hunk 1]
```diff
@@ -275,14 +275,14 @@ export default function Proposal({
             Proposal passed and executed successfully
           </div>
         ) : (
-          <div className="mt-10 flex justify-evenly">
+          <>
```
Comment (Lucianosc) [inline apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx:278]: Don't think there is a need to wrap the component with a fragment, maybe you can return only the component

**Review by Lucianosc — CHANGES_REQUESTED** [review]

File: apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx [hunk 2]
```diff
@@ -281,6 +281,7 @@ export default function Proposal({
               thresholdPct={thresholdPct}
               proposalSupportPct={totalSupportPct}
               isSignalingType={isSignalingType}
+              proposalId={proposalIdNumber as string}
```
Comment (Corantin) [inline apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx:284]: no type assertions :p

**Review by Corantin — APPROVED** [review]
Love it 🔥

File: apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx (hunk 1 above)
Comment (Mati0x) [reply apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx:278]: Good catch!

**Review by Corantin — APPROVED** [review]

**Comment by Corantin** [comment]
image
@Mati0x where you able to put a flag in the tooltip finally ?

**Comment by Mati0x** [comment]
> image @Mati0x where you able to put a flag in the tooltip finally ?

not yet goss

**Comment by Mati0x** [comment]
> > image @Mati0x where you able to put a flag in the tooltip finally ?
> 
> not yet goss, It is a minor detail since threshold is very clear to user. It is in TODO list before we lunch

**Pull request merged**